contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_import_order.py

```diff
@@ -48,7 +49,7 @@
 
 
 def strip_newline(stmt):
-  return textwrap.dedent('\n'.join(filter(None, stmt.splitlines())))
+  return textwrap.dedent('\n'.join([_f for _f in stmt.splitlines() if _f]))
```

Comment:
I think this can be converted to a generator. Simply remove the `[]`, and we should get minor performance enhancements.

Reply:
Think we'd need to wrap it in () but correct we don't need the intermediate list